spec/models/acts_as_customizable/calculated_value_spec.rb

```diff
@@ -381,4 +382,194 @@ def custom_field_values(*) = nil
       end
     end
   end
+
+  describe "#calculate_custom_fields error handling" do
+    def expect_calculated_value_error(calc_val_cf, project, error_code, message_part = nil)
+      error = calc_val_cf.first_calculation_error(project)
+      expect(error&.error_code).to eq(error_code)
+
+      if message_part
+        expect(calculated_value_error_msg(error)).to include(message_part)
+      end
+    end
+
+    let(:project) { create(:project) }
+    let(:enabled_custom_fields) { {} }
```

Comment:
If I understand correctly, it is used only to create values, but not enable custom field in the project